k6/src/modules/auth/model/User.js

```diff
@@ -0,0 +1,8 @@
+class User {
+    constructor() {
+        this._cookieName = undefined;
+        this._cookieValue = undefined;
```

Comment:
It's a common practice to use underscore prefix to mark class properties as private, but in this case, these are public. I would suggest removing the underscores.

Reply:
Also as I see these two properties are not used at all.

Reply:
Done, thanks!